tests/test_db.py

```diff
@@ -439,30 +412,29 @@ def test_unset_patchset_pending(self, mock_sql, mock_get_sourceid,
 
     @mock.patch('logging.debug')
     @mock.patch('sktm.db.SktDb._SktDb__get_baselineresult')
-    @mock.patch('sktm.db.SktDb._SktDb__commit_testrun')
     @mock.patch('sktm.db.SktDb._SktDb__get_repoid')
     @mock.patch('sktm.db.sqlite3')
     def test_update_baseline_new(self, mock_sql, mock_get_repoid,
-                                 mock_commit_testrun, mock_get_baselineresult,
-                                 mock_log):
+                                 mock_get_baselineresult, mock_log):
```

Comment:
Could you please extract this and following changes into a separate commit, as they don't seem to be related?

Reply:
The `commit_testrun` method is being removed, so it can't be mocked and needs to be removed from the test signature too, and the fact that we now have one more DB interaction within the `update_baseline` method needs to be reflected in the tests so the commit doesn't cause a regression in the tests.

Reply:
I see, thank you. Perhaps a note in the commit message would have helped me to understand what's going on and avoid unnecessary round-trip for explanations :)

Reply:
We have that "commit shouldn't break the build" rule, so I thought fixing the tests is implicit :) Will mention it in the future